### ISSUE TITLE
Force destroy a node?

### ISSUE THREAD

**Issue by SouDescolado** [geringverdien/TeamFishnet#1]
Hello, is it possible to force destroy a player node with the scripts? I want to delete that script kiddie bot, since it prevents people from joining

**Comment by SouDescolado** [comment]
The intention would be to delete a node as a host, to 'unblock' the other players that do not want to install mods or anything

**Comment by geringverdien** [comment]
I cant check right now because im on my phone but iirc the bot only uses the network api to connect to the server and im not sure if that even creates a player node, you should decompile the game following BlueberryWolf's modding guide and see how connecting to a server is handled, what type of nodes are instantiated/required to be connected and from there you could figure out if theres a way to forcefully disconnect a client from your end

**Comment by geringverdien** [comment]
Yeah im planning to make it draggable, as for other features i dont really know what to add except for maybe opening locally stored files
